### ISSUE TITLE
Start (Urgent)

### ISSUE THREAD

**Issue by YoungPup** [Paulowarren31/Supreme-Bot#9]
Hi, I'm too afraid to use the "wait" so if I press start like 5 seconds before 11, will it keep refreshing and looking for the item/s? Or do I have to wait until the items are there? Hope you can answer this before 11

**Comment by Paulowarren31** [comment]
it looks for the items. I just updated the item list with new items.

**Comment by YoungPup** [comment]
So it's fine if I start it like 5 sec before?

**Comment by Paulowarren31** [comment]
It will keep refreshing the store until it finds what you select so yes

**Comment by ttokkyo222** [comment]
i don't see an updated items list.

**Comment by Paulowarren31** [comment]
Try refreshing your cache.

**Comment by cjkenney** [comment]
How do I find new item codes?

**Comment by Paulowarren31** [comment]
Just updated them. UK cannot get item codes now because they are now the
earliest drop.

**Comment by ttokkyo222** [comment]
Are some of the new item codes supposed to have "spaces" in it??

**Comment by Paulowarren31** [comment]
Yeah they give some spaces. Just make sure to leave the code as is when you
add it. Let me know how the drop goes for you.

**Comment by ttokkyo222** [comment]
ok, will do... thanks. Are you purchasing anything today paulo?

**Comment by Paulowarren31** [comment]
I think I'll go for a backpack.

**Comment by Paulowarren31** [comment]
https://gyazo.com/d52cf3798ba483fe7d1028a90748a57c


got it! Captcha never showed up. Weird.

**Comment by YoungPup** [comment]
I got my stuff too, but the bot really lagged once it first dropped, and I had to manually press "Add to cart" and the CVV I had to put in manually because I think they adjusted the auto fill.

**Comment by Paulowarren31** [comment]
Same here. I will update right now for new cvv field

**Comment by Paulowarren31** [comment]
hmm I am testing it now and cvv is filling in correctly. Not sure what was
going on.

**Comment by YoungPup** [comment]
Nice! Thanks :) 

I also think the CC # needed to be clicked on, because the format was different. It was all one line until you clicked on it, then it separated into 4 groups of 4 numbers (VISA)

**Comment by ttokkyo222** [comment]
yea they changed it, i was testing the bot 10mins before launch and it worked fine.

**Comment by ttokkyo222** [comment]
would future implementation of proxy support be possible with this bot?
